benchmark/profile_generation.py

```diff
@@ -244,36 +275,57 @@ class ProfileResult:
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Regression Test')
-    parser.add_argument('--model-path',
+    parser.add_argument('model_path',
```

Comment:
this change is a BC and the doc should be updated:
https://github.com/InternLM/lmdeploy/tree/main/benchmark#profile-without-dataset

BTW, maybe we could use `fire` to replace `argparse`. Because other scripts in benchmark use `fire` cli package.

Reply:
yes, you can check the docs: https://github.com/google/python-fire/blob/master/docs/guide.md#argument-parsing
eg: `--concurrency [1,16,32,64]`

Reply:
In the case of list, I think it is not as simple as argparse, e.g., the input data has to be embraced by `[]` and there cannot be any whitespace in the list